contracts/modules/SuperMinterV1_1.sol

```diff
@@ -100,6 +100,23 @@ contract SuperMinterV1_1 is ISuperMinterV1_1, EIP712 {
             ")"
         );
 
+    /**
+     * @dev For EIP-712 presave signature digest calculation.
+     */
+    bytes32 public constant PRESAVE_TYPEHASH =
```

Comment:
rename presave and all usages to

platformAirdrop

way more clear what it is